execution/executor/transfer_test.go

```diff
@@ -44,10 +44,14 @@ func (td *testData) checkTotalCoin(t *testing.T, fee int64) {
 	assert.Equal(t, total+fee, int64(21000000*1e9))
 }
 
-func (td *testData) randomAmountAndFee(max int64) (int64, int64) {
-	amt := td.RandInt64(max / 2)
+func (td *testData) randomAmountAndFee(min int64, max int64) (int64, int64) {
+	amt := td.RandInt64NonZero(max)
+	for amt < min {
+		amt = td.RandInt64NonZero(max)
+	}
+
 	fee := int64(float64(amt) * td.sandbox.Params().FeeFraction)
-	return amt, fee
+	return fee, amt
```

Comment:
why??

Reply:
that was return them reverse I think
but I can change order of them when call it